src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -1286,4 +1289,11 @@ Friend Module CompilationUtils
         Dim result = If(namespacesAndTypesOnly, model.LookupNamespacesAndTypes(position, container), model.LookupSymbols(position, container))
         Return result.Select(Function(s) s.Name).Distinct().ToList()
     End Function
+
+    Private Sub ValidateIOperations(createCompilation As Func(Of Compilation))
```

Comment:
>ValidateIOperations [](start = 16, length = 19)

Can we move this function to CompilationExtensions.cs? I would be very convenient if TEST_IOPERATION_INTERFACE was used in just one file, it would be very easy to enable verification and run tests in VS.